Library/Homebrew/requirements.rb

```diff
@@ -134,7 +146,8 @@ def initialize(tags)
   def java_version
     args = %w[/usr/libexec/java_home --failfast]
```

Comment:
I know this is probably my bad code but it feels weird to have `/usr/libexec/java_home` as part of `args`; perhaps split that out to the `system` invocation.